src/sagemaker/serializers.py

```diff
@@ -357,3 +356,38 @@ def serialize(self, data):
             return data.read()
 
         raise ValueError("Unable to handle input format: %s" % type(data))
+
+
+class DataSerializer(SimpleBaseSerializer):
+    """Serialize data in any file by extracting raw bytes from the file."""
+
+    def __init__(self, content_type="file-path/raw-bytes"):
+        """Initialize a ``DataSerializer`` instance.
+
+        Args:
+            content_type (str): The MIME type to signal to the inference endpoint when sending
+                request data (default: "file-path/raw-bytes").
+        """
+        super(DataSerializer, self).__init__(content_type=content_type)
+
+    def serialize(self, data):
+        """Serialize file data to a raw bytes.
+
+        Args:
+            data (object): Data to be serialized. The data can be a string
+                representing file-path or the raw bytes from a file.
+        Returns:
+            raw-bytes: The data serialized as raw-bytes from the input.
+        """
+        if isinstance(data, str):
+            try:
+                dataFile = open(data, "rb")
+                dataFileInfo = dataFile.read()
+                dataFile.close()
```

Comment:
Wouldn't it have made more sense to the context manager for reading the file, to be safer about error handling when `.read()` fails? 
Also shouldn't be the naming schema for variables `snake_case` rather than `camelCase`? Similar to all other [Serializer](https://github.com/aws/sagemaker-python-sdk/blob/5c64e6ce399f24e7f02cce8f028ce4b032843e2c/src/sagemaker/serializers.py#L110)

Reply:
As this PR is already merged, added a new PR with the suggested changes here: https://github.com/aws/sagemaker-python-sdk/pull/2962